test/opt/ccp_test.cpp

```diff
@@ -581,6 +581,66 @@ TEST_F(CCPTest, SkipSpecConstantInstrucitons) {
   auto res = SinglePassRunToBinary<opt::CCPPass>(spv_asm, true);
   EXPECT_EQ(std::get<1>(res), opt::Pass::Status::SuccessWithoutChange);
 }
+
+TEST_F(CCPTest, UpdateSubsequentPhisToVarying) {
```

Comment:
This test does not seem to have any CHECKs?

Reply:
Correct its testing the pass succeeds with no change.